kernel/packages/shared/meta/types.ts

```diff
@@ -63,5 +63,6 @@ export type CommsConfig = {
 
 export enum FeatureFlags {
   WEARABLES_V2 = 'wearables_v2',
+  QUESTS_ENABLED = 'quests_enabled',
```

Comment:
`QUESTS_ENABLED` the url param flag like this would be `ENABLE_QUESTS_ENABLED` 😅

Reply:
Pato has a point, probably adding the `enabled` to a feature flag may be an anti-naming-pattern `quests` may suffice

Reply:
Sure, gonna change it